### PR TITLE
MODINVOSTO-51 - Schema updates for Batch Voucher Export Config API

### PR DESCRIPTION
**Pull request by craigmcnally**
## Purpose
When implementing the schemas for this we missed the export_config_collection schema.  Also included here are several other minor changes such as adjustments to the export_configuration->uploadURI regex pattern

## Approach
* Add the missing collection schema/example
* Adjust the uploadURI pattern as needed

#### TODOS and Open Questions
- [ ] Merge this before https://github.com/folio-org/mod-invoice-storage/pull/59

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [X] Examples exist for all schemas
  - [X] Descriptions exist for all schema properties
  - [X] All schemas pass raml-cop linting
- Does this introduce breaking changes?
  - [ ] Were there any schema changes?
  - [X] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: mod-invoice-storage/schemas/export_configuration.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
     "uploadURI": {
       "description": "URI batch vouchers should be uploaded to.  Limited to FTP initially",
       "type": "string",
-      "pattern": "^ftp://([\\.a-z0-9]+)(/([\\./a-z0-9]?)+)?$"
+      "pattern": "^ftp://([\\.\\_\\-a-z0-9]+)(/([\\./\\-\\_\\.a-z0-9]?)+)?$"
```
Comment (pautau-s) [inline mod-invoice-storage/schemas/export_configuration.json]: URI from example do not pass this regex,

Comment (craigmcnally) [reply mod-invoice-storage/schemas/export_configuration.json]: 🤔 They does for me... how are you testing it?

Comment (craigmcnally) [reply mod-invoice-storage/schemas/export_configuration.json]:
```
package cmcnally.sandbox;

import java.util.regex.Pattern;

public class App {
  public static void main(String[] args) {

    Pattern p = Pattern.compile("^ftp://([\\.\\_\\-a-z0-9]+)(/([\\./\\-\\_\\.a-z0-9]?)+)?$");
    System.out.println(p.matcher("ftp://ftp.amherst-lib.edu/invoices/").matches());
  }
}
```
returns true for me

Comment (pautau-s) [reply mod-invoice-storage/schemas/export_configuration.json]: 🤔it works for me as well. I must have missed something last time.

**Pull request merged**